src/network/NetworkManagerPrimary.cc

```diff
@@ -17,6 +17,11 @@
 
 #include "NetworkManagerPrimary.hh"
 
+#include <gz/msgs/world_stats.pb.h>
+#include <gz/msgs/serialized_map.pb.h>
+#include "gz/sim/private_msgs/peer_control.pb.h"
+#include "gz/sim/private_msgs/simulation_step.pb.h"
```

Comment:
do you need `private_msgs/performer_affinity` here or is it ok that it's already included transitively by `simulation_step.pb.h`?